__tests__/postService.test.js

```diff
@@ -94,10 +129,83 @@ describe('createReply test', () => {
         const response = await createReply(username, text, id);
         let added = false;
         mockDatabase.forEach((post) => {
-            if(post.itemID == id && post.replies.length > 0){
+            if (post.itemID == id && post.replies.length > 0) {
                 added = true;
             }
         });
         expect(added).toBeTruthy();
     });
+});
+
+describe('getPostById', () => {
+    it('Successful get post', async () => {
+        const id = mockPost1.itemID;
+        const expectedDescription = mockPost1.description;
+
+        const post = await getPostById(id);
+
+        expect(post.itemID).toEqual(id);
+        expect(post.description).toEqual(expectedDescription);
+    });
+
+    it('Throws if post not found', async () => {
+        const id = "FakeID";
+        let error;
+        const expectedStatus = 500;
```

Comment:
I think we need the expectedStatus to be 400 instead of 500 since the getPostById in the PostService throws a status code of 400 and not 500 if the post is not found.